project/scripts/release

```diff
@@ -366,6 +366,7 @@ if [ $dry_run ]; then
   echodry "  sbt deployRsync"
   echodry "  rsync -rlpvz --chmod=Dg+ws,Fg+w --exclude ${release_dir}/downloads --exclude ${release_dir}/docs ${release_dir}/ ${publish_path}/"
 else
+  important tar -czf ${release_dir}/offline.tar.gz ${release_dir}/*
```

Comment:
isn't `zip` format more accepted? That can be done with sbt as we do in the `release-2.4` branch:
https://github.com/akka/akka/blob/release-2.4/project/Dist.scala#L112

Reply:
Either are fine I think.

Reply:
I don’t think tar.gz is widly used on Windows?